core/src/main/res/values/attrs.xml

```diff
@@ -72,4 +72,7 @@
             <enum name="minimum" value="2"/>
         </attr>
     </declare-styleable>
+
+    <declare-styleable name="ChapterSeekBar" />
```

Comment:
Is this needed?

Reply:
Oh, you're right. It's not. Thanks! (Forgot to remove it)